src/components/ui/buttons/Button.tsx

```diff
@@ -0,0 +1,149 @@
+import * as React from "react"
+import { cva, type VariantProps } from "class-variance-authority"
+import { Slot } from "@radix-ui/react-slot"
+
+import { BaseLink, type LinkProps } from "@/components/Link"
+
+import { cn } from "@/lib/utils/cn"
+import { type MatomoEventOptions, trackCustomEvent } from "@/lib/utils/matomo"
+import { scrollIntoView } from "@/lib/utils/scrollIntoView"
+
+const buttonVariants = cva(
+  "inline-flex gap-2 items-center justify-center rounded border-[1px] border-solid border-current text-primary transition focus-visible:outline focus-visible:outline-4 focus-visible:outline-primary-hover focus-visible:-outline-offset-1 disabled:text-disabled disabled:pointer-events-none hover:text-primary-hover [&[data-secondary='true']]:text-body",
+  {
+    variants: {
+      variant: {
+        solid:
+          "!text-background bg-primary border-transparent disabled:bg-disabled disabled:text-background hover:text-background hover:bg-primary-hover hover:shadow-button-hover active:shadow-none",
```

Comment:
is it necessary the `!`?
```suggestion
          "text-background bg-primary border-transparent disabled:bg-disabled disabled:text-background hover:text-background hover:bg-primary-hover hover:shadow-button-hover active:shadow-none",
```

Reply:
@pettinarip Checkout the override story and the `ButtonLink`. Without marking this style as `!important` the text color gets lost.

This still has to do with specificity between Tailwind and Chakra, where I shouldn't have it marked like this once Chakra is not a part of this.

Not going to know until this PR is merged and checked with updates you did to the links.

Reply:
Got it, ok.